tips/git.md

```diff
@@ -0,0 +1,51 @@
+# `git` Related Tips
+
+This file contains various tips and tricks for using the `git` CLI (and GitHub too!)
+
+## Glossary
+
+Some helpful `git` and GitHub related terms.
+
+- `repository`: A repository is like a folder that holds a project's files, but unlike other folders a repository is tracked for changes by `git`.

```

Comment:
Might want to mention that `repo` is a short form of `repository` or vice versa for future convinience